includes/classes/admin/mapping/field-types/acf.php

```diff
@@ -52,8 +53,13 @@ public function underscore_template( View $view ) {
 			}, $groupIds ) );
 
 			// Prepare and execute query to get all fields for all groups
-			$fields_query   = "SELECT * FROM {$wpdb->posts} WHERE post_type = 'acf-field' AND post_content LIKE '%repeater%' AND post_parent IN ($groupIdPlaceholders)";
-			$fields_results = $wpdb->get_results( $wpdb->prepare( $fields_query, $groupIds ) );
+			$fields_results = $wpdb->get_results(
+				$wpdb->prepare(
+					"SELECT * FROM {$wpdb->posts} WHERE post_type = 'acf-field' AND post_content LIKE %s AND post_parent IN ($groupIdPlaceholders)",
+					'%' . $wpdb->esc_like('repeater') . '%',
```

Comment:
I don't get this change, wasn't `repeater` just a hardcoded string anyway, way does it need to be escaped?

Reply:
It's a suggestion from the plugin checker.

**ERROR**: WordPress.DB.PreparedSQLPlaceholders.LikeWildcardsInQuery
**Message**: SQL wildcards for a LIKE query should be passed in through a replacement parameter. Found: LIKE '%repeater%'.